.github/actions/push-release-commit/action.yml

```diff
@@ -151,33 +140,25 @@ runs:
         
         gh api --method POST /repos/:owner/:repo/git/refs -f ref="refs/heads/$TEMPORARY_BRANCH" -f sha="$GITHUB_SHA"
 
-        # Parse modules and iterate over them, committing API files and collecting their details
-        # in a json object for later use
-        IFS=',' read -ra MOD_ARRAY <<< "$MODULES"
-
-        API_FILES_JSON="["
-
-        for mod in "${MOD_ARRAY[@]}"; do
-          MOD=$(echo "${mod}" | xargs)
-          API_FILE_PATH=$(git diff-tree --no-commit-id --name-only -r "$RELEASE_TAG" | grep "${MOD}/${API_FILE}")
-          API_FILE_INITIAL_SHA=$(git rev-parse "$GITHUB_REF":"$API_FILE_PATH")
-          API_FILE_RELEASE_SHA=$(git rev-parse "$RELEASE_TAG":"$API_FILE_PATH")
-          API_FILE_RELEASE_CONTENT=$(git cat-file blob "$RELEASE_TAG":"$API_FILE_PATH" | base64 -w0)
-          API_FILE_POST_RELEASE_CONTENT=$(git cat-file blob unsigned/"$GITHUB_REF_NAME":"$API_FILE_PATH" | base64 -w0)
-
-          # Commit the API file changes for module
-          gh api --method PUT /repos/:owner/:repo/contents/"$API_FILE_PATH" \
-            --field branch="$TEMPORARY_BRANCH" \
-            --field message="Update public API file for module $MOD for $RELEASE_TAG" \
-            --field sha="$API_FILE_INITIAL_SHA" \
-            --field content="$API_FILE_RELEASE_CONTENT" --jq '.commit.sha'
-          
-
-          API_FILES_JSON="${API_FILES_JSON}{\"module\":\"$MOD\",\"api_file_path\":\"$API_FILE_PATH\",\"api_file_release_sha\":\"$API_FILE_RELEASE_SHA\",\"api_file_post_release_content\":\"$API_FILE_POST_RELEASE_CONTENT\"},"
-        done
-
-        # Remove trailing comma and close JSON array
-        API_FILES_JSON="${API_FILES_JSON%,}]"
+        # Only commit API files for modules in API_UPDATED_MODULES
+        if [ -n "$API_UPDATED_MODULES" ]; then
+          IFS=',' read -ra UPDATED_MODS <<< "$API_UPDATED_MODULES"
+          for mod in "${UPDATED_MODS[@]}"; do
+            MOD=$(echo "${mod}" | xargs)
```

Comment:
I know this was here before, but what's the purpose of this line?

Reply:
Yep, equivalent of `.trim()`